packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md

```diff
@@ -0,0 +1,148 @@
+
+**What you need**
+
+- An Okta developer org configured for a **Stay signed in** use case
+- Access to the [My Account Sessions API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/Sessions/)
+- [Postman client](https://www.getpostman.com/downloads/) to set up `userPersistentCookie`. See [Get Started with the Okta APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+
+**Sample code**
+
+[Request example](#request-example)
+
+---
+
+## About **Stay signed in**
+
+**Stay signed in** lets users establish an Okta session that continues across browsers and devices. When a user selects this option during authentication, they won't be prompted again for MFA for the amount of time defined in your global session policy.
+
+There are two ways to present the **Stay signed in** option to your users. By default, it's displayed on the Sign-In Widget when users enter their credentials. You can also configure it so that the prompt appears after users authenticate. This option is recommended for orgs that use delegated authentication or identity providers, because their users may bypass the credential entry page of the Sign-In Widget.
+
+**Stay signed in** is only recommended for users on private devices or networks. Users who suspect that their credentials have been compromised can end all sessions in their account settings.
+
+See [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in) for more details.
+
+
+## Enable the userPersistentCookie parameter
+
+Enable the `usePersistentCookie` option in the API if you want users to stay signed in across browser sessions.
+
+### About Single Sign-On (SSO) cookies
+
+SSO allows users to authenticate once and access multiple resources without being prompted for more credentials. Okta supports both session and persistent SSO:
+
+Session SSO: Session SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications during a particular session. However, if a particular session ends, the user is prompted for their credentials again.
+
+Persistent SSO: Persistent SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications for as long as the persistent SSO cookie is valid.
+
+The difference between persistent SSO and session SSO is that you can maintain persistent SSO across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session across different sessions, you need to enable persistent SSO.
+
+### Enable the parameter
+
+To enable persistent SSO for **Stay signed in**, you need to update the [`usePersistentCookie`](/docs/reference/api/policy/#signon-action-object) parameter in the appropriate global session policy. These steps use the Okta [Policy Postman collection](/docs/reference/postman-collections/) to enable the parameter.
+
+Before you begin, be sure to:
```

Comment:
```suggestion
Before you begin, complete the following steps:
```

Reply:
Globalization team requests that all stems function as complete sentences (they don't read the list items as complementary to the stem fragment). This is just something we've started doing on prod side. Will be required when dev docs start getting translated.